docs/install-manual.md

```diff
@@ -16,6 +16,21 @@ This page teaches you to deploy a high-availability Docker Swarm cluster.
 Although the example installation uses the Amazon Web Services (AWS) platform,
 you can deploy an equivalent Docker Swarm cluster on many other platforms. In this example, you do the following:
 
+
+<!-- TOC depth:2 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Build a Swarm cluster for production](#build-a-swarm-cluster-for-production)
+	- [Prerequisites](#prerequisites)
+	- [Step 1. Add network security rules](#step-1-add-network-security-rules)
+	- [Step 2. Create your instances](#step-2-create-your-instances)
+	- [Step 3. Install Engine on each node](#step-3-install-engine-on-each-node)
+	- [Step 4. Set up a discovery backend](#step-4-set-up-a-discovery-backend)
+	- [Step 5. Create Swarm cluster](#step-5-create-swarm-cluster)
+	- [Step 6. Communicate with the Swarm](#step-6-communicate-with-the-swarm)
+	- [Step 7. Test Swarm failover](#step-7-test-swarm-failover)
+	- [Additional Resources](#additional-resources)
+<!-- /TOC -->
+
```

Comment:
The link to Mike's blog was not resolving.  The blog seems down. I'm removing that for now.